cli/helpers/sca_code_scanner.py

```diff
@@ -12,13 +12,13 @@
 
 
 def run_pre_scan_actions(context: click.Context, documents_to_scan: List[Document], is_git_diff: bool = False):
-    is_monitor = context.params.get('monitor')
+    is_monitor_action = context.obj.get('monitor')
```

Comment:
since its set only in repository scan, it can be null in case of path scan. 
please change to context.obj.get('monitor', false)